src/com/kodilla/controls/Connector.java

```diff
@@ -1,7 +1,19 @@
 package com.kodilla.controls;
 
+import javafx.stage.Stage;
+
+import java.util.List;
+
 public interface Connector {
 
-    void userMouseClicked(String userMark, int rowClicker, int columnClicked);
+    interface Presenter {
+        void userMouseClicked(String userMark, int rowClicker, int columnClicked);
+
+        void setView(com.kodilla.ui.Board view, Stage primaryStage);
```

Comment:
Dodaj import, jak pojawia sie package to nie ładnie to wygląda

Reply:
done!